ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -310,6 +312,261 @@ describe('a4a_config', () => {
       }
     });
   });
+  it('should serve Delayed Fetch to unlaunched DoubleClick filler',
+    () => {
+      toggleExperiment(win, 'a4aFastFetchDoubleclickLaunched', false, true);
+      element.setAttribute('type', 'doubleclick');
+      let externalBranches, internalBranches;
+      if (isExperimentOn(win, 'a4aFastFetchDoubleclickLaunched')) {
```

Comment:
Good point. The net difference between the two approaches is that as-is, it validates that toggleExperiment() and isExperimentOn() work properly...but these tests aren't the places for doing so.